src/controllers/comment.controller.js

```diff
@@ -3,29 +3,33 @@ import models from '../db/models';
 export default class commentController {
 
     static async createComment(req,res) {
-        try{
-    
+  
           const commenterId = req.user.id; 
           const { comment } = req.body;
           const { requestId } = req.query;
           
-              const newComment = await models.Comment.create({
+              await models.Comment.create({
                 commenterId,
                 requestId,
                 comment,
               });
-
-              if(newComment){
               return res.status(201).json({
                   status: 201,
                   message: `comment successfully added`,
-              })
-            }
-        } 
-        catch(error){
-            return res.status(500).json({
-                error: error,
-                })
-            }
+              });
       }
+
+      static async deleteComment(req, res) {
+
+        const { commentId } = req.query;
```

Comment:
why having an empty line here Please add some ESLint roles to avoid that.

Reply:
@kagabof  The eslint was introduced and applied here is rule, which is there by default but was not activated. 

<img width="106" alt="Screen Shot 2020-03-09 at 10 52 02" src="https://user-images.githubusercontent.com/48021034/76197022-0874ac80-61f4-11ea-9aa0-7bdec2fa143e.png">

It will now only accept two empty lines